pkg/plugin/service.go

```diff
@@ -32,8 +33,8 @@ import (
 //     separate process and communicates with it via gRPC. These plugins are
 //     compiled independently of Conduit and can be included at runtime.
 type registry interface {
-	NewDispenser(logger log.CtxLogger, name FullName) (Dispenser, error)
-	List() map[FullName]Specification
+	NewDispenser(logger log.CtxLogger, name FullName) (connector.Dispenser, error)
+	List() map[FullName]connector.Specification
 }
 
 type Service struct {
```

Comment:
IIUC, this plugin service is supposed to return all plugins (connectors + processors)?

Reply:
Yes, although I'm not married to this idea, I'm open to splitting it.

Reply:
I guess it depends on the usage... Do we know for sure we'll need to return all plugins somewhere, UI maybe? My guess is that it'll be always or most of the time just one or the other.

Reply:
It will probably always be either one or the other, however the service will look the same for both. I might make the service itself generic (similar to how the registry interface is generic) and instantiate one for processor and one for connector plugins, that could be nicer. In a later PR, I'll merge this as is for now.